webapp/donateSrc/app/components/Locator/index.jsx

```diff
@@ -26,21 +26,32 @@ const Locator = ({ gender, orientation }) => {
     setPointerY(relY);
     setPointerD('block')
 
-    console.log('mole X: ', relX - (parentEl.current.offsetWidth / 2));
-    console.log('mole Y: ', relY - (parentEl.current.offsetHeight / 2));
+    // move the origin of the coordinates to be relative to the center
+    // of the body image.
+    const bodyX = relX - (parentEl.current.offsetWidth / 2);
+    const bodyY = relY - (parentEl.current.offsetHeight / 2);
+
+    // console.log('mole X: ', bodyX);
+    // console.log('mole Y: ', bodyY);
+
+    const bodyLocationField = document.getElementById('body_location');
+    const bodyImageField = document.getElementById('body_image');
+
+    if (bodyLocationField) bodyLocationField.value = `${bodyX}, ${bodyY}`;
+    if (bodyImageField) bodyImageField.value = `${gender}-${orientation}`;
```

Comment:
We need to add a body_image field (or something similar) to the mole model. The coordinates only make sense if we know which image they are based on.
i.e. a mole at `50, -10` on FemaleRight is very different from `50, -10` on MaleFront.

Reply:
So this line doesn't work yet, but it doesn't break as it is here.